doc-site/.vitepress/theme/components/ChangeThemaSelect.vue

```diff
@@ -0,0 +1,150 @@
+<template>
+  <div
+    tabindex="0"
```

Comment:
Har du vurdert å bruke nve-dropdown eller nve-select i steder for å bygge komponenten fra grunnen av?

Reply:
Det har jeg, men jeg tenkte at omstyling så at den passer resten av komponentene i nav baren vil ta for mye tid, så valgte å lage egen style på den.